tasks/build-tachometer-comment.cjs

```diff
@@ -16,11 +16,19 @@ const fs = require('fs');
 const prettyBytes = require('pretty-bytes');
 
 const getTachometerResults = () => {
-    const results = [];
-    for (const result of globby.sync(`./tach-results.*.json`)) {
+    const results = {
+        chrome: [],
+        firefox: [],
+    };
+    for (const result of globby.sync(`./tach-results.chrome.*.json`)) {
+        const file = fs.readFileSync(result, 'utf8');
+        const json = JSON.parse(file);
+        results.chrome.push(json.benchmarks);
+    }
```

Comment:
With two browsers this duplication isn't a big deal, but there is the option to wrap this in a function like `getTachBrowserResults(pattern)` so that `getTachometerResults` would just be:

```ts
return {
  chrome: getTachBrowserResults('./tach-results.chrome.*.json'),
  firefox: getTachBrowserResults('./tach-results.firefox.*.json'),
}
```

Reply:
Went with a little smarter workflow on:
```js
for (const result of globby.sync(`./tach-results.*.json`)) {
```
, sound good?

Reply:
Yep!